clover.rb

```diff
@@ -16,8 +16,7 @@ def self.freeze
   end
 
   route do |r|
-    subdomain = r.host.split(".").first
-    if subdomain == "api"
+    if r.host.start_with?("api.")
```

Comment:
Should we use header_matcher or is it too much for just single usage?

Reply:
I think it's probably overkill for just this usage.